features/list-projects.feature

```diff
@@ -0,0 +1,51 @@
+Feature: List dependencies
+
```

Comment:
Could you a small sentence, maybe the contents of the original issue here to describe why we have this feature?

Reply:
Thanks!